lms/services/digest/_models.py

```diff
@@ -0,0 +1,76 @@
+from dataclasses import dataclass
+from functools import cached_property
+from typing import List
+
+
+@dataclass
+class HUser:
+    """A class representing H users."""
+
+    h_userid: str
+    """The H userid like 'acct:name@lms.example.com'."""
+
+    display_name: str
+    """A human readable display name."""
+
+    email: str
+    """Email for the user."""
+
+
+@dataclass
+class HCourse:
+    """A class representing courses as groups in H."""
```

Comment:
This is a bit of a fib, but I think it gets the idea across. We are talking about a group in H not a grouping in LMS.